engine/graphics/src/webgpu/graphics_webgpu.cpp

```diff
@@ -800,76 +832,82 @@ void WebGPUContext::instanceRequestAdapterCallback(WGPURequestAdapterStatus stat
     }
 }
 
-bool WebGPUContext::initialize(const ContextParams& params)
+static bool initializeWebGPUContext(WebGPUContext* context, const ContextParams& params)
```

Comment:
Instead of `WebGPUCotnext` ctor.

Reply:
Will do as a next piece of changes (I also want to remove `GetMaxElementsIndices` across all graphics adapters)